releng/org.eclipse.emfcloud.modelserver.releng.p2/pom.xml

```diff
@@ -77,9 +77,9 @@
                      <appArgLine>-application org.eclipse.ant.core.antRunner -buildfile packaging-p2composite.ant p2.composite.add -Dsite.label=${p2.site.label} -Dproject.build.directory=${project.build.directory} -DunqualifiedVersion=${unqualifiedVersion} -DbuildQualifier=${buildQualifier} -Dsoftware.download.area=${rsync.local.dir}</appArgLine>
                      <repositories>
                         <repository>
-                           <id>2021-06</id>
+                           <id>2022-03</id>
```

Comment:
This should reflect the actual used targetplatform application, which was not properly updated last time, I just aligned it here.

About 2022-12, we cannot update yet as with 2022-06 they removed Jetty 9 and switched entirely to Jetty 10. We should do this separately as this will be a rather big change (see also #234).
There is also an update regarding the maven plugin for the targetplatform, which is now more stable and got additional features, so I tend to go away from orbit then with the next update.